### PR TITLE
Add GitHub Actions configuration

### LINKED ISSUE #87: Release build for other platforms
Since Wafter is not platform-dependent, building for other OS should not be a problem.

### PR DESCRIPTION
**Pull request by minhducsun2002**
Closes #87.

### CODE REVIEW

**Comment by dungwinux** [comment]
There's versioning problem, please see `src/config/version.js`

**Comment by minhducsun2002** [comment]
Done.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Node.js CI
+
+on: push
```
Comment (dungwinux) [inline .github/workflows/main.yml]: Please apply the CI only on master like below:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Comment (minhducsun2002) [reply .github/workflows/main.yml]: Why? Shouldn't all commits be tested before merging?

Comment (dungwinux) [reply .github/workflows/main.yml]: Well, yes, but it should be limited to master branch only.

Comment (minhducsun2002) [reply .github/workflows/main.yml]: What is the problem with testing all branches?

**Pull request merged**